CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Install [Spack](https://spack.io) by default in cluster user's home directory.
 
 **CHANGES**
+- Migrate NFS exports and associated data for intra-cluster shared storage to external AWS EFS filesystems attached to the cluster stack.
```

Comment:
I'd specify that we are moving the NFS export from the head node root volume. Not only to be more specific on tech standpoint but because this has an impact on cost and on the required size of that volume (moving from EBS root volume to EFS).

Reply:
sounds good!